### PR TITLE
DW-3689 create dataworks-admin-utils repo

### PR DESCRIPTION
**Pull request by mark-r-m**
DW-3689 create dataworks-admin-utils repo

### CODE REVIEW

File: dataworks-admin-utils.tf [hunk 1]
```diff
@@ -0,0 +1,34 @@
+resource "github_repository" "dataworks-admin-utils" {
+  name        = "dataworks-admin-utils"
+  description = "Contains dataworks administrative utilities"
```
Comment (dan-hill2802) [inline dataworks-admin-utils.tf]: Should be `DataWorks`
comment not request

File: dataworks-admin-utils.tf [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+}
+
+resource "github_branch_protection" "dataworks_admin_utils_master" {
+  branch         = "${github_repository.dataworks-admin-utils.default_branch}"
+  repository     = "${github_repository.dataworks-admin-utils.name}"
+  enforce_admins = false
+
+  required_status_checks {
+    strict = true
+    contexts = ["concourse-ci/dataworks-admin-utils-pr"]
```
Comment (dan-hill2802) [inline dataworks-admin-utils.tf]: this won't exist, so remove this line

Comment (mark-r-m) [reply dataworks-admin-utils.tf]: The readme instructions don't say to remove that part? And there should be a PR job in the end? of os this a two-phase approach?

File: dataworks-admin-utils.tf (hunk 1 above)
Comment (mark-r-m) [reply dataworks-admin-utils.tf]: Done, ta :-)

File: README.md [hunk 3]
```diff
@@ -10,6 +10,8 @@ Manage GitHub team and repository configuration for DataWorks
 1. Update the `github_repository` resource name, and its `name` and `description` attributes. *Do not* make any other changes.
 1. Update the 3 references to the `github_repository` resource so their names match the new name you used in step 2.
 1. Make sure all `example` entries are replaced with new repo name as used in step 3.
+1. The `required_status_checks` stanza should only be kept for Terraform repos. 
+   1. If the new repo will not contain terraform, remove the `required_status_checks` stanza.
```
Comment (dan-hill2802) [inline README.md]: The `required_status_checks` should always exist, but it only needs `contexts` for Terraform repos

File: dataworks-admin-utils.tf (hunk 2 above)
Comment (dan-hill2802) [reply dataworks-admin-utils.tf]: the `contexts` need removing/emptying

File: repository.tf.sample [hunk 4]
```diff
@@ -22,6 +22,7 @@ resource "github_branch_protection" "example_master" {
   repository     = "${github_repository.example.name}"
   enforce_admins = false
 
+  # This stanza should only be kept for Terraform repos. Application code will run through circle.
```
Comment (dan-hill2802) [inline repository.tf.sample]: stanza still needed, but contexts should be removed

**Pull request merged**